cpansa/CPANSA-HTTP-Daemon.yml

```diff
@@ -0,0 +1,33 @@
+---
+- affected_versions: '<6.15'
```

Comment:
HTTP-Daemon hasn't been released yet

Reply:
I took out that version. Now it's just null.